buildscripts/kokoro/linux_artifacts.sh

```diff
@@ -33,3 +33,14 @@ popd
 readonly MVN_ARTIFACT_DIR="${MVN_ARTIFACT_DIR:-$GRPC_JAVA_DIR/mvn-artifacts}"
 mkdir -p "$MVN_ARTIFACT_DIR"
 cp -r "$LOCAL_MVN_TEMP"/* "$MVN_ARTIFACT_DIR"/
+
+# for aarch64 platform
+apt-get install -y wget autoconf automake libtool g++-aarch64-linux-gnu g++ make
```

Comment:
Looks like this needs a 'sudo' before it. I'll tweak some of this and push an update.